test/perf/shootout/k_nucleotide.jl

```diff
@@ -46,7 +46,7 @@ function sorted_array(m::Dict{AbstractString, Int})
         kn[i] = KNuc(elem...)
         i += 1
     end
-    sort(kn)
+    sort!(kn)
```

Comment:
Since this code is used to track performance, it's probably better to leave this change out, to maintain continuity.  (I doubt it's that important, though, so I'm open to arguments otherwise...)